e2e-tests/wiremock/src/main/java/com/foo/wiremock/WireMockApplication.java

```diff
@@ -0,0 +1,14 @@
+package com.foo.wiremock;
+
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+public class WireMockApplication {
```

Comment:
might create an E2E that contains two services, ie, `X` and `Y`. then one endpoint of `X` is to make a call to an endpoint of `Y`.